src/Tests/Unit/BaseIsracardAmexBranches.test.ts

```
@@ -0,0 +1,316 @@
+
+  it('logs ProxyRequestHandler URLs via response callback', async () => {
+    mockValidate('1');
+    mockLogin('1');
+    mockAccounts();
+    mockTxns();
+    const page = CREATE_MOCK_PAGE();
+    MOCK_CONTEXT.newPage.mockResolvedValue(page);
+    await new TestAmexScraper().scrape(CREDS);
+    const calls = page.on.mock.calls as [
+      string,
+      (r: { url: () => string; status: () => number }) => string,
```

ESLint

[error] 'r' is defined but never used.

(no-unused-vars)